e2e/metricbeat_test.go

```diff
@@ -143,39 +142,18 @@ func MetricbeatFeatureContext(s *godog.Suite) {
 
 	s.BeforeSuite(func() {
 		log.Debug("Before Metricbeat Suite...")
-		serviceManager := services.NewServiceManager()
 
 		env := map[string]string{
 			"stackVersion": stackVersion,
 		}
 
-		err := serviceManager.RunCompose(true, []string{"metricbeat"}, env)
-		if err != nil {
-			log.WithFields(log.Fields{
-				"stack": "metricbeat",
-			}).Error("Could not run the stack.")
-		}
-
-		minutesToBeHealthy := 3 * time.Minute
-		healthy, err := waitForElasticsearch(minutesToBeHealthy, "metricbeat")
-		if !healthy {
-			log.WithFields(log.Fields{
-				"error":   err,
-				"minutes": minutesToBeHealthy,
-			}).Error("The Elasticsearch cluster could not get the healthy status")
-		}
+		startRuntimeDependencies("metricbeat", env, 3)
```

Comment:
The third argument here is a `time.Duration`. As it stands, we're using a magic number here, `3`, for this argument. The nice thing about this being a `time.Duration` is that we can instead use `3 * time.Minute` here, which makes it less of a magic number and conveys more easily what this argument is.

Reply:
Love this suggestion! In fact I had in mind to add an env var to be able to modify that at CI time